Bugzilla.pm

```diff
@@ -256,12 +261,12 @@ sub extensions {
 }
 
 sub cgi {
-    return $_[0]->request_cache->{cgi} ||= new Bugzilla::CGI();
+    return request_cache->{cgi} ||= new Bugzilla::CGI();
 }
 
 sub input_params {
     my ($class, $params) = @_;
```

Comment:
$class no longer used? Could it be dropped?

Reply:
class is still used here.

Reply:
Oh .. I am not usedto the notion of return-if, you are right